packages/native/src/index.tsx

```diff
@@ -0,0 +1,8 @@
+import { NavigationContainerRef } from '@navigation-ex/core';
+import useBackButton from './useBackButton';
+
+export { useBackButton };
+
+export default function useNativeIntegration(ref: NavigationContainerRef) {
```

Comment:
move the hook to separate file since we only re-export in `index` files everywhere